README.md

```diff
@@ -1,187 +1,81 @@
-<h1 align="center">Arranger</h1>
-<p align="center">Generate and manage your own genomic data portal.</p>
+# Arranger - Search Query API and Portal UI Microservice
 
-<p align="center"><a href="http://www.overture.bio/products/arranger" target="_blank"><img alt="Release Candidate" src="http://www.overture.bio/img/progress-horizontal-RC.svg" width="320" /></a></p>
+[<img hspace="5" src="https://img.shields.io/badge/chat-on--slack-blue?style=for-the-badge">](http://slack.overture.bio)
+[<img hspace="5" src="https://img.shields.io/badge/License-gpl--v3.0-blue?style=for-the-badge">](https://github.com/overture-stack/arranger/blob/develop/LICENSE)
+[<img hspace="5" src="https://img.shields.io/badge/Code%20of%20Conduct-2.1-blue?style=for-the-badge">](code_of_conduct.md)
 
-[![Slack](http://slack.overture.bio/badge.svg)](http://slack.overture.bio)
+<div>
+<img align="right" width="120vw" src="icon-arranger.png" alt="arranger-logo"/>
+</div>
 
-Develop (Edge): [![Build Status](https://jenkins.qa.cancercollaboratory.org/buildStatus/icon?job=Overture.bio%2Farranger%2Fdevelop)](https://jenkins.qa.cancercollaboratory.org/job/Overture.bio/job/arranger/job/develop/)
+Arranger is a data-agnostic search API that interfaces between an Elasticsearch index and pre-built portal UI components. This allows you to quickly create and customize functional data portal UI's, enabling researchers with a custom GUI to query data, build cohorts and export these filtered data sets for further analysis and interpretation.
 
-Master (Release): [![Build Status](https://jenkins.qa.cancercollaboratory.org/buildStatus/icon?job=Overture.bio%2Farranger%2Fmaster)](https://jenkins.qa.cancercollaboratory.org/job/Overture.bio/job/arranger/job/master/)
+Arranger consists of three elements:
 
-> **Note**: Arranger is undergoing refactoring work, for what will become version 3+. While we do not foresee many dramatic breaking changes, an _upgrade guide_ is already in the works. Current users of v2 may look at the `legacy` branch meanwhile, where we will continue fixing newly reported bugs until further notice.
->
-> You may need to add the `--legacy-peer-deps` flag when integrating our modules into your apps. Along with the rewrite, we're steadily updating the internal dependencies so this isn't necessary.
+- **Arranger Search**, a search API layered above an Elasticsearch cluster
```

Comment:
This could be better labelled as the "Arranger Server". We could also include links to SQON documentation, etc.